core/translate/main_loop.rs

```diff
@@ -401,10 +401,10 @@ pub fn open_loop(
 
                     program.resolve_label(loop_start, program.offset());
                     // TODO: We are currently only handling ascending indexes.
-                    // For conditions like index_key > 10, we have already seeked to the first key greater than 10, and can just scan forward.
+                    // For conditions like index_key > 10, we have already sought to the first key greater than 10, and can just scan forward.
```

Comment:
i mean, it's technically correct, but... maybe this should be `performed a seek` 😅